docs/en/end-user/application.md

```diff
@@ -68,6 +68,87 @@ spec:
           - '1000'
 ```
 
+### Specify Definition Revision in Application
```

Comment:
This doc should be moved to platform team doc and simplified, an example could be: https://github.com/resouer/kubevela/blob/97037b2efe10bee3791fbe3b985be8fabc90049d/docs/en/platform-engineers/definition-revision.md

Also, please fix this issue: https://github.com/oam-dev/kubevela/issues/1637

Reply:
got it